### PR TITLE
Spanish Translation 

### PR DESCRIPTION
**Pull request by rafa-gould**

### CODE REVIEW

File: locale/es.po [hunk 1]
```diff
@@ -295,6 +295,8 @@ msgstr "Valores del array deben ser bytes individuales."
 #: supervisor/shared/safe_mode.c
 msgid "Attempted heap allocation when MicroPython VM not running.\n"
 msgstr ""
+"Se intentó la asignación del montón cuando la VM de"
```
Comment (sabas1080) [inline locale/es.po]: Monto suena raro o no, a que se refiere? Tal vez sea mejor dejar esa palabra en ingles o no?

Comment (rafa-gould) [reply locale/es.po]: Por supuesto yo lo cambiare!

**Comment by tannewt** [comment]
@erikriver or @C47D do you have time to review? I'd love to get this into the next release candidate but want to get it out today.

**Comment by sabas1080** [comment]
ready @tannewt , thanks @rafa-gould

**Comment by tannewt** [comment]
Thank you @sabas1080 and @rafa-gould

**Pull request merged**